examples/cli/readme.md

```diff
@@ -1 +1,619 @@
-placeholder
\ No newline at end of file
+# Learning the `benchit` CLI
+
+This document is a tutorial for exploring the different features of the `benchit` command line interface (CLI). You can learn the details of those features in the [Tools User Guide](https://github.com/groq/mlagility/blob/main/docs/tools_user_guide.md) and learn about their implementation in the [Code Organization Guide](https://github.com/groq/mlagility/blob/main/docs/code.md).
+
+We've created this tutorial document because `benchit` is a CLI that benchmarks the contents of `.py` scripts. So all of the `.py` scripts in the `examples/cli/scripts` directory are meant to be fed into `benchit` to demonstrate some specific functionality.
+
+Once you've familiarized yourself with these features, head over to the [`models` directory](https://github.com/groq/mlagility/tree/main/models) to learn how to use `benchit` with real world machine learning models.
+
+# Table of Contents
+
+TODO
+
+# Tutorials
+
+All of the tutorials assume that your command line is in the same location as this readme file (`examples/cli`).
```

Comment:
"current working directory" instead of "command line"?

Reply:
good call